src/layers/domain/request_models/v1.py

```diff
@@ -72,12 +73,23 @@ class SpineMhsQuestionnaireRsponse(BaseModel):
     __root__: list[dict] = Field(min_items=1, max_items=1)
 
 
+class SpineAsQuestionnaireResponse(BaseModel):
+    __root__: list[dict] = Field(min_items=1, max_items=1)
+
+
 class CreateMhsDeviceIncomingParams(BaseModel, extra=Extra.forbid):
     questionnaire_responses: dict[
         Literal[QuestionnaireInstance.SPINE_MHS], SpineMhsQuestionnaireRsponse
     ] = Field(...)
 
 
+class CreateAsDeviceIncomingParams(BaseModel, extra=Extra.forbid):
+    name: str = AS_DEVICE_NAME
```

Comment:
Remove name param from here so the user can't overwrite it and add this at the create device step for consistency :)